jaxrs/src/main/java/com/facebook/airlift/jaxrs/JaxrsModule.java

```diff
@@ -49,7 +50,7 @@ public JaxrsModule(boolean requireExplicitBindings)
     }
 
     @Override
-    public void configure(Binder binder)
+    protected void setup(Binder binder)
```

Comment:
Wondering why this method should be protected.

Reply:
The goal here is to register `JerseyAuthorizationFilter` only when we configure the `authEnabled` as `true`.

In order to read the config, we need to make this module extends [`AbstractConfigurationAwareModule`](https://github.com/prestodb/airlift/blob/6daede1956c25c6df1aa5ae4d7473a55cd319976/configuration/src/main/java/com/facebook/airlift/configuration/AbstractConfigurationAwareModule.java#L70), in which the method signature is `protected abstract void setup(Binder binder)`.

This module has a similar usage: [com.facebook.presto.verifier.source.SourceQueryModule](https://github.com/prestodb/presto/blob/master/presto-verifier/src/main/java/com/facebook/presto/verifier/source/SourceQueryModule.java)